### ISSUE TITLE
Cannot see the send or download invoice icons - have these been removed?

### ISSUE THREAD

**Issue by PercyP** [YetiForceCompany/YetiForceCRM#3028]
Hi,

I am just doing a test invoice and after completing, I wanted to download it to see how it looked, however, I cannot see any way of doing this (there used to be an icon for downloading).  I also tried on your demo site and cannot see them there either. I realise you are making loads of changes so maybe its done a different way now or have they been removed?

Percy

**Comment by vovpff** [comment]
@percyp you must made PDF template at admin section to Invoice module. There is no icon if no templates

**Comment by PercyP** [comment]
Hi Vovpff,

thank you!

Is there any documentation on how to do these things? I have searched bu cannot find anything, including at yetiforce.com. There is docs for some things, but not this.

**Comment by vovpff** [comment]
@PercyP let's check this section Software configuration > System tools > PDF
https://gitdeveloper.yetiforce.com/index.php?module=PDF&parent=Settings&view=List&block=4&fieldid=35
There is no problem to made header and footer of page. Editor are simple. for editing product list you need experiwnxe and PHP knowleges

**Comment by PercyP** [comment]
In the previous version 2.xxx we didn't have to create one of these, there was a basic one already installed. I just want something like that. I looked at the old yetiforce invoice but the old yetiforce doesn't seem to provide an opportunity to view the pdf templates to see how they are layed out. Do we really need PHP experience to create invoice templates?

**Comment by PercyP** [comment]
Update. 
I found where the templates are stored in the old one, and tried copying and pasting into the new one but it doesn't work. Then I discovered the PDF module has now been replaced :(  so it does appear to need building from scratch. Can yetiforce not put some standard templates in its shop for us to download/install if its not included in the system? Without ability to print send an invoice we cannot use it sadly.

P

**Comment by PercyP** [comment]
I finally figured it out. It is fairly simple to create a new template. My problem was I was only adding the labels and not the values. This was because of an issue with Google Chrome not copying the values text or allowing me to paste it. So when I clicked on the value buttons nothing was happening and not knowing any different, assumed that was how it was meant to work.

I restarted my pc, cleared all browser cache and it started working as it should.

Creating the template is relatively easy now its working properly (and thanks to @vovpff for his video clip on how to do it: https://github.com/YetiForceCompany/YetiForceCRM/issues/3037

Percy
